bin/webpack-dev-server.js

```diff
@@ -93,6 +94,15 @@ const config = require('webpack-cli/bin/convert-argv')(yargs, argv, {
 // we should use portfinder.
 const DEFAULT_PORT = 8080;
 
+// Try to find unused port and listen on it for 3 times,
+// if port is not specified in options.
+// Because NaN == null is false, defaultTo fails if parseInt returns NaN
+// so the tryParseInt function is introduced to handle NaN
+const defaultPortRetry = defaultTo(
+  tryParseInt(process.env.DEFAULT_PORT_RETRY),
```

Comment:
IMO: I think we should leave `process.env.DEFAULT_PORT_RETRY` as a document.

Reply:
@hiroppy can you create issue in webpack docs repo?

Reply:
I think `DEFAULT_PORT_RETRY` should be added in `devServer` because currently, we don' use `process.env`. see https://webpack.js.org/configuration/dev-server/

@evilebottnawi what do you think?

Reply:
@hiroppy yep, after merge we need open issue here https://github.com/webpack/webpack.js.org :+1: